CLAUDE.md

```
@@ -815,7 +815,85 @@ then "draft" else "published" end` (실측: `v1.0.1 -> draft`, `v1.0.0 -> publis
   코드와 git 로그가 이미 "무엇을 했는지"는 말하므로, 여기 그것만 적으면 쓸 이유가 없다.
 - **작업을 마칠 때마다 커밋한다.** 컨텍스트가 날아가도 커밋 로그가 남으면 복구된다.
 
-## 모바일 (착수 전) — 빌드는 정말 필요할 때만
+## 모바일 — `apps/mobile` (진행 중, #102)
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)